fluss-client/src/main/java/com/alibaba/fluss/client/admin/internal/AdminApiDriver.java

```diff
@@ -0,0 +1,193 @@
+                    .lookup(Collections.unmodifiableSet(lookupKeys))
+                    .thenAcceptAsync(
+                            lookupResult -> {
+                                completeLookup(lookupResult.mappedKeys);
+                                completeLookupExceptionally(lookupResult.failedKeys);
+                                try {
+                                    Thread.sleep(RETRY_DELAY_MS);
+                                } catch (InterruptedException e) {
+                                    throw new RuntimeException(e);
+                                }
+                                // maybe send requests again if not finished.
+                                maybeSendRequests();
+                            });
+        }
+
+        for (Map.Entry<Integer, Set<K>> entry : fulfillmentMap.entrySet()) {
+            handler.handle(entry.getKey(), entry.getValue())
+                    .thenAcceptAsync(
+                            apiResult -> {
+                                complete(apiResult.completedKeys);
+                                completeExceptionally(apiResult.failedKeys);
+                                retryLookup(apiResult.unmappedKeys);
+                                // maybe send requests again if not finished.
+                                maybeSendRequests();
+                            });
+        }
+    }
+
+    private void retryLookup(Collection<K> keys) {
+        keys.forEach(this::unmap);
+    }
+
+    private void completeLookup(Map<K, Integer> brokerIdMapping) {
+        if (!brokerIdMapping.isEmpty()) {
+            future.completeLookup(brokerIdMapping);
+            brokerIdMapping.forEach(this::map);
+        }
+    }
+
+    private void completeLookupExceptionally(Map<K, Throwable> errors) {
+        if (!errors.isEmpty()) {
+            future.completeLookupExceptionally(errors);
+            clear(errors.keySet());
+        }
+    }
+
+    /**
+     * Complete the future associated with the given key. After this is called, all keys will be
+     * taken out of both the Lookup and Fulfillment stages so that request are not retried.
+     */
+    private void complete(Map<K, V> values) {
+        if (!values.isEmpty()) {
+            future.complete(values);
+            clear(values.keySet());
+        }
+    }
+
+    /**
+     * Complete the future associated with the given key exceptionally. After is called, the key
+     * will be taken out of both the Lookup and Fulfillment stages so that request are not retried.
+     */
+    private void completeExceptionally(Map<K, Throwable> errors) {
+        if (!errors.isEmpty()) {
+            future.completeExceptionally(errors);
+            clear(errors.keySet());
+        }
+    }
+
+    /**
+     * Associate a key with a brokerId. This is called after a response in the Lookup stage reveals
+     * the mapping (e.g. when the `FindCoordinator` tells us the group coordinator for a specific
```

Comment:
Can you check all of the docs, we don't have a operation named  `FindCoordinator` and `group coordinator`.